### PR TITLE
chore: refactored hooks name

### PR DESCRIPTION
**Pull request by Kibo-TriptiV**

### CODE REVIEW

File: components/checkout/StandardShippingStep/StandardShippingStep.tsx [hunk 1]
```diff
@@ -78,8 +78,8 @@ const StandardShippingStep = (props: ShippingProps) => {
     setStepStatusComplete,
     setStepStatusIncomplete,
   } = useCheckoutStepContext()
-  const updateCheckoutShippingInfo = useUpdateCheckoutShippingInfoMutation()
-  const { data: shippingMethods } = useShippingMethodsQueries(
+  const updateCheckoutShippingInfo = useUpdateOrderShippingInfo()
```
Comment (kibo-chandradeeptalaha) [inline components/checkout/StandardShippingStep/StandardShippingStep.tsx]: change this to updateOrderShippingInfo

File: components/layout/AppHeader/Checkout/CheckoutHeader.tsx [hunk 2]
```diff
@@ -23,12 +23,12 @@ const CheckoutHeader = ({ isMultiShipEnabled }: { isMultiShipEnabled: boolean })
...
   const router = useRouter()
   const { checkoutId } = router.query
-  const { data: multishipCheckout } = useMultiShipCheckoutQueries({
+  const { data: multishipCheckout } = useGetCurrentCheckout({
     checkoutId: checkoutId as string,
     isMultiShip: isMultiShipEnabled,
   })
 
-  const { data: checkout } = useCheckoutQueries({
+  const { data: checkout } = useGetCurrentOrder({
```
Comment (kibo-chandradeeptalaha) [inline components/layout/AppHeader/Checkout/CheckoutHeader.tsx]: change `{ data : checkout }` to `{ data : order }`

File: components/page-templates/CartTemplate/CartTemplate.tsx [hunk 3]
```diff
@@ -43,17 +43,17 @@ export interface CartTemplateProps {
...
   const { t } = useTranslation('common')
   const theme = useTheme()
   const isMobileViewport = useMediaQuery(theme.breakpoints.down('md'))
   const router = useRouter()
-  const { createFromCart } = useCreateFromCartMutation()
-  const { createMultiShipCheckoutFromCart } = useCreateMultiShipCheckoutFromCartMutation()
-  const { updateCartItemQuantity } = useUpdateCartItemQuantityMutation()
-  const { removeCartItem } = useRemoveCartItemMutation()
-  const { updateCartItem } = useUpdateCartItemMutation()
+  const { createFromCart } = useInitiateOrder()
```
Comment (kibo-chandradeeptalaha) [inline components/page-templates/CartTemplate/CartTemplate.tsx]: change createFromCart to initiateOrder

File: components/page-templates/CartTemplate/CartTemplate.tsx [hunk 4]
```diff
@@ -43,17 +43,17 @@ export interface CartTemplateProps {
...
+  const { createMultiShipCheckoutFromCart } = useInitiateCheckout()
```
Comment (kibo-chandradeeptalaha) [inline components/page-templates/CartTemplate/CartTemplate.tsx]: change this to initiate checkout

File: components/page-templates/MultiShipCheckoutTemplate/MultiShipCheckoutTemplate.tsx [hunk 5]
```diff
@@ -134,8 +134,8 @@ const MultiShipCheckoutTemplate = (props: MultiShipCheckoutProps) => {
   }
 
   // Payment Step
-  const updateMultiShipCheckoutPaymentAction = useUpdateMultiShipCheckoutPaymentActionMutation()
-  const createMultiShipCheckoutPaymentAction = useCreateMultiShipCheckoutPaymentActionMutation()
+  const updateMultiShipCheckoutPaymentAction = useVoidCheckoutPayment()
```
Comment (kibo-chandradeeptalaha) [inline components/page-templates/MultiShipCheckoutTemplate/MultiShipCheckoutTemplate.tsx]: change this to voidCheckoutPayment

File: components/page-templates/MultiShipCheckoutTemplate/MultiShipCheckoutTemplate.tsx [hunk 6]
```diff
@@ -134,8 +134,8 @@ const MultiShipCheckoutTemplate = (props: MultiShipCheckoutProps) => {
...
+  const createMultiShipCheckoutPaymentAction = useAddCheckoutPayment()
```
Comment (kibo-chandradeeptalaha) [inline components/page-templates/MultiShipCheckoutTemplate/MultiShipCheckoutTemplate.tsx]: change this to addCheckoutPayment

File: components/page-templates/MultiShipCheckoutTemplate/MultiShipCheckoutTemplate.tsx [hunk 7]
```diff
@@ -157,7 +157,7 @@ const MultiShipCheckoutTemplate = (props: MultiShipCheckoutProps) => {
   }
 
   // Review Step
-  const createOrder = useCreateMultiShipCheckoutMutation()
+  const createOrder = useCreateCheckout()
```
Comment (kibo-chandradeeptalaha) [inline components/page-templates/MultiShipCheckoutTemplate/MultiShipCheckoutTemplate.tsx]: change this to createCheckout

File: components/page-templates/StandardShipCheckoutTemplate/StandardShipCheckoutTemplate.tsx [hunk 8]
```diff
@@ -33,20 +33,16 @@ const StandardShipCheckoutTemplate = (props: StandardShipCheckoutProps) => {
   const [promoError, setPromoError] = useState<string>('')
   const { checkoutId } = router.query
 
-  const { data: checkout } = useCheckoutQueries({
+  const { data: checkout } = useGetCurrentOrder({
```
Comment (kibo-chandradeeptalaha) [inline components/page-templates/StandardShipCheckoutTemplate/StandardShipCheckoutTemplate.tsx]: change this to order

File: components/page-templates/StandardShipCheckoutTemplate/StandardShipCheckoutTemplate.tsx [hunk 9]
```diff
@@ -87,9 +83,9 @@ const StandardShipCheckoutTemplate = (props: StandardShipCheckoutProps) => {
 
   // Payment Step
 
-  const updateOrderPaymentAction = useUpdateOrderPaymentActionMutation()
-  const createOrderPaymentMethod = useCreateCheckoutPaymentMethodMutation()
-  const updateCheckoutBillingInfo = useUpdateCheckoutBillingInfoMutation()
+  const updateOrderPaymentAction = useVoidOrderPayment()
```
Comment (kibo-chandradeeptalaha) [inline components/page-templates/StandardShipCheckoutTemplate/StandardShipCheckoutTemplate.tsx]: change this to voidOrderPayment

File: components/page-templates/StandardShipCheckoutTemplate/StandardShipCheckoutTemplate.tsx [hunk 10]
```diff
@@ -87,9 +83,9 @@ const StandardShipCheckoutTemplate = (props: StandardShipCheckoutProps) => {
...
+  const createOrderPaymentMethod = useAddOrderPaymentInfo()
```
Comment (kibo-chandradeeptalaha) [inline components/page-templates/StandardShipCheckoutTemplate/StandardShipCheckoutTemplate.tsx]: change this to addOrderPayment

File: components/page-templates/StandardShipCheckoutTemplate/StandardShipCheckoutTemplate.tsx [hunk 11]
```diff
@@ -87,9 +83,9 @@ const StandardShipCheckoutTemplate = (props: StandardShipCheckoutProps) => {
...
+  const updateCheckoutBillingInfo = useUpdateOrderBillingInfo()
```
Comment (kibo-chandradeeptalaha) [inline components/page-templates/StandardShipCheckoutTemplate/StandardShipCheckoutTemplate.tsx]: change this to updateOrderBillingInfo

File: hooks/mutations/standardCheckout/useUpdateOrderShippingInfo/useUpdateCheckoutShippingInfoMutation.ts
Comment (kibo-chandradeeptalaha) [inline hooks/mutations/standardCheckout/useUpdateOrderShippingInfo/useUpdateCheckoutShippingInfoMutation.ts:1]: change file name

**Review by kibo-chandradeeptalaha — CHANGES_REQUESTED** [review]
Please take a look

**Review by kibo-chandradeeptalaha — APPROVED** [review]

**Review by kibo-sushant — APPROVED** [review]

**Review by kibo-kevinwatts — APPROVED** [review]

**Pull request merged**